src/js/resize.js

```diff
@@ -0,0 +1,22 @@
+import Split from 'split.js'
+
+const PAGE_WIDTH = 960
+
+const isSplited = () => document.getElementsByClassName('gutter').length > 0
+const isBigScreen = () => window.innerWidth > PAGE_WIDTH
+const main = document.getElementById('main')
+const gutter = () => document.getElementsByClassName('gutter')[0]
+
+const makeGutter = () => {Split(['#left_pane', '#right_pane'], {
+    gutterSize: 8,
+    cursor: 'col-resize'
```

Comment:
Cursor should be either `col-resize` or `row-resize` depending on your orientation

Reply:
How do you mean? If we destroy the gutter, why would we change the cursor?